src/components/atoms/GalleryImage/styles.tsx

```diff
@@ -8,15 +8,23 @@ export const StyleGalleryImageContainer = styled.div`
     width: 10000px;
 `;
 
-export const StyleGalleryImage = styled(SimpleImg)`
-  img {
-    margin: 0;
-    padding: 0;
-  }
+export const StyleGalleryImage = styled.img`
+    position: absolute;
+    left: -10000%;
+    right: -10000%;
+    top: -10000%;
+    bottom: -10000%;
+    margin: auto auto;
+    min-width: 1000%;
+    min-height: 1000%;
+    -webkit-transform: scale(.1);
```

Comment:
in your css file never write `prefix` like `webkit-` 

https://www.reddit.com/r/css/comments/2yjmaj/stop_writing_vendor_prefixes_autoprefixer_does/

Note: There are just some instances where you'll need to write a prefix, for example for scrollbars.